README.md

```diff
@@ -205,6 +223,15 @@ The Lock/Unlock trait is used for anything that can lock and unlock, such as doo
 - Lock Command: A device command used to lock the device.  Should not require any parameters.  Maps to `lock` by default.
 - Lock Command: A device command used to unlock the device.  Should not require any parameters.  Maps to `unlock` by default.
 
+### Media State
+
+The Media State trait is used for reporting the current playback and activity state of a media device.  Query command is unknown at the is time.  The trait has the following configuration parameters:
+
+- Support Activity State: Should be set if this device can report the current activity state.  Defaults to 'false'.
+- Support Playback State: Should be set if this device can report the current playback state.  Defaults to 'false'.
+- Activity State Attribute: The device attribute used to query the current activity state of the device.  Maps to the `activityState` attribute by default.
+- Playback State Attribute: The device attribute used to query the current playback state of the device.  Maps to the `playbackState` attribute by default.
```

Comment:
This setting should only be available if "Support Playback State" is true.  It should probably default to either `status` to match Hubitat's "MusicPlayer" capability or `transportStatus` to match Hubitat's "MediaTransport" capability.  I'm not sure which capability is used more often.  My only point of reference is that the Sonos integration uses "MusicPlayer" and not "MediaTransport", so maybe that's the more-often used media player capability?

Reply:
Fixed.  Defaulted to Sonos "status".